lib/engine/game/base.rb

```diff
@@ -506,9 +506,9 @@ def inspect
 
       def result
         @players
-          .sort_by(&:value)
-          .reverse
           .map { |p| [p.name, player_value(p)] }
+          .sort_by { |_, v| v }
+          .reverse
```

Comment:
It is a bit surprising that reverse (or better yet, reverse!) is the fastest way of doing this.
https://stackoverflow.com/questions/2642182/how-to-sort-an-array-in-descending-order-in-ruby